src/containers/LoginPage.jsx

```diff
@@ -1,20 +1,44 @@
-// external libs
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { replace } from 'react-router-redux';
 
-// my libs (utils, actions)
+import { userLogin } from '../actions';
+import { getLoggingIn } from '../selectors/login';
 
-// external components
+import Login from '../components/Login.jsx';
 
-// my components
+@connect(mapStateToProps, { replace, userLogin })
+export default class LoginPage extends Component {
+    static propTypes = {
+        loggingIn: PropTypes.bool.isRequired,
+        replace: PropTypes.func.isRequired,
+        userLogin: PropTypes.func.isRequired
+    };
 
-// styles
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.loggingIn) {
+            return;
+        }
 
-// constants
+        // TODO: Add error handling
+        // TODO: Add router state handling
 
-export default class LoginPage extends Component {
-    render () {
+        this.props.replace({
+            pathname: '/totalscore'
+        });
+    }
+
+    handleClick = login => this.props.userLogin(login);
+
+    render() {
         return (
-            <h1>Login page</h1>
+            <Login onClick={this.handleClick} />
```

Comment:
Why onClick? It is better to have properly named callbacks. They need to express what is going on. 

For example in this case it's better to name it onLogin or smthng